### DISCUSSION TITLE
不知道为啥最近编译的一直nss打不开

### DISCUSSION THREAD

**Discussion by wutaijieing** [Boos4721/openwrt#48] [Bugs/问题反馈]
流量分载  未运行
log：
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.455148] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.461575] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.469437] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.475144] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.483082] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.488966] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.495708] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.503174] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.509488] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED
Fri Dec 10 22:55:14 2021 kern.err kernel: [  785.517619] wlan1: NSS TX failed with error[8]: NSS_TX_FAILURE_NOT_ENABLED

[config.zip](https://github.com/Boos4721/openwrt/files/7693716/config.zip)

**Comment by wutaijieing** [comment]
貌似kmod-qca-nss-ecm没有加载，这怎么破

**Comment by Boos4721** [comment]
`insmod kmod-qca-nss-ecm`, and clean your router config.
